src/core/expression/qgsexpression.h

```diff
@@ -596,6 +596,17 @@ class CORE_EXPORT QgsExpression
      */
     static QString helpText( QString name );
 
+    /**
+     * Add help string for a custom function/variable.
```

Comment:
Isn't this only for variables?

Reply:
Hum, I thought it could work for custom python function but it seems not.
I tried quickly :
```python
@qgsfunction(args='auto', group='Test')
def test_function_help(value1, value2, feature, parent):
    return value1 + value2
QgsExpression.addHelpText('test_function_help', 'My help string')
```

Ok, I will rename with `addVariableHelpText`